routes/contactUs/helpers.js

```diff
@@ -0,0 +1,26 @@
+import emailJs from "emailjs";
+
+export const contactUsmail = async (req, res) => {
+    try {
+        const server = emailJs.server.connect({
```

Comment:
You should destructure `process.env` for `EMAIL_HOST, EMAIL_PASSWORD, USER_EMAIL...etc`